server/src/app.ts

```diff
@@ -147,7 +139,7 @@ if (app.get("env") === "development") {
   app.use(errorHandler());
 } else {
   app.use((error: any, req: Request, res: Response, _next: NextFunction) => {
-    console.error("ERROR:", error);
+    logger.error("ERROR:", error);
```

Comment:
See above about redundant level info and different arguments between `console.error` and `logger.error`.